src/background/browser-name.js

```diff
@@ -4,6 +4,7 @@
  * @return {'chrome'|'firefox'}
  */
 export default function browserName() {
+  // @ts-expect-error - `browser` is missing from types
```

Comment:
Is `ts-ignore` a better option?
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error